selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginTest.java

```diff
@@ -231,12 +296,29 @@ private void configureTypeOfProject() {
     wizard.waitCreateProjectWizardFormIsClosed();
   }
 
+  private void checkoutToBranch(String branchName) {
+    menu.runCommand(GIT, BRANCHES);
+    git.waitBranchInTheList(branchName);
+    git.selectBranchAndClickCheckoutBtn(branchName);
+    git.waitGitCompareBranchFormIsClosed();
+  }
+
   private void openFileAndChangeContent(String filePath, String text) throws Exception {
     projectExplorer.openItemByPath(filePath);
     editor.waitActive();
     testProjectServiceClient.updateFile(testWorkspace.getId(), filePath, text);
   }
 
+  /** check the target branches in the pull header */
+  private void checkBranchNamesOnGitHubPage() {
+    String pullHeaderText =
+        format(
+            "%s wants to merge 1 commit into %s\n" + "from %s",
+            gitHubUsername, BASE_BRANCH_NAME, DEV_BRANCH_NAME);
+    seleniumWebDriverHelper.waitTextContains(
+        By.xpath("//div[@class='TableObject-item TableObject-item--primary']"), pullHeaderText);
```

Comment:
It's better to name `//div[@class='TableObject-item TableObject-item--primary']` somehow to understand it's propose.